test/minimallist/generator_test.cljc

```diff
@@ -385,9 +385,23 @@
   (let [model (-> (h/map [:a fn-int?])
                   (h/with-optional-entries [:b fn-string?]))
         sample (tcg/sample (gen model) 100)]
-    (is (and (every? (partial valid? model) sample)
-             (some (fn [element] (contains? element :b)) sample)
-             (some (fn [element] (not (contains? element :b))) sample))))
+    (is (every? (partial valid? model) sample))
+    (is (every? (fn [element] (contains? element :a)) sample))
+    (is (some (fn [element] (contains? element :b)) sample))
+    (is (some (fn [element] (not (contains? element :b))) sample))
+    (is (some (fn [element] (and (contains? element :a)
+                                 (not (contains? element :b)))) sample)))
```

Comment:
Those 2 lines can be deleted because the same condition is tested above:
```
    (is (every? (fn [element] (contains? element :a)) sample))
    (is (some (fn [element] (contains? element :b)) sample))
```